backend/.env

```diff
@@ -1,3 +1,6 @@
 PORT=3000
 IPFS_URL_ADD='http://localhost:8080/ipfs/add'
```

Comment:
don't check in the .env file

Reply:
I accidentally committed it before I placed it in .gitignore. Once saved, it seems that it can not be ignored, but I will see to uncheck when I commit changes.